### ISSUE TITLE
SDK headers not installed by default in Xcode 10.2

### ISSUE THREAD

**Issue by zhihaoy** [microsoft/vcpkg#6950]
```
CMake Error at /Users/travis/Library/Caches/vcpkg/Source/vcpkg/scripts/buildsystems/vcpkg.cmake:266 (_find_package):
  Found package configuration file:
    /Users/travis/Library/Caches/vcpkg/Source/vcpkg/installed/x64-osx/share/tbb/TBBConfig.cmake
  but it set TBB_FOUND to FALSE so package "TBB" is considered to be NOT
  FOUND.
```

**Comment by zhihaoy** [comment]
I'm using an older version; needs reproduce.

**Comment by zhihaoy** [comment]
TBB U5-1 ~ U7, all same issue.

**Comment by zhihaoy** [comment]
```
(venv) Traviss-Mac:tbb travis$ cat config-x64-osx-rel-out.log
-- The CXX compiler identification is AppleClang 10.0.1.10010046
-- Check for working CXX compiler: /Applications/Xcode-10.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
-- Check for working CXX compiler: /Applications/Xcode-10.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Building Intel TBB: /usr/bin/make -j2 tbb_build_prefix=tbb_cmake_build_subdir tbb_build_dir=/Users/travis/Library/Caches/vcpkg/Source/vcpkg/buildtrees/tbb/x64-osx-rel/tbb_cmake_build compiler=clang extra_inc=big_iron.inc
-- Building is unsuccessful (2): /Users/travis/Library/Caches/vcpkg/Source/vcpkg/buildtrees/tbb/x64-osx-rel/src/./build/Makefile.tbb:28: CONFIG: cfg=debug arch=intel64 compiler=clang target=macos runtime=cc10.0.1_os10.14.4
In file included from /Users/travis/Library/Caches/vcpkg/Source/vcpkg/buildtrees/tbb/x64-osx-rel/src/./src/tbb/concurrent_hash_map.cpp:17:
In file included from /Users/travis/Library/Caches/vcpkg/Source/vcpkg/buildtrees/tbb/x64-osx-rel/src/./include/tbb/concurrent_hash_map.h:20:
In file included from /Users/travis/Library/Caches/vcpkg/Source/vcpkg/buildtrees/tbb/x64-osx-rel/src/./include/tbb/tbb_stddef.h:431:
In file included from /Applications/Xcode-10.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/memory:654:
In file included from /Applications/Xcode-10.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/typeinfo:61:
In file included from /Applications/Xcode-10.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/exception:82:
In file included from /Applications/Xcode-10.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/cstdlib:86:
/Applications/Xcode-10.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/stdlib.h:94:15: fatal error: 'stdlib.h' file not found
#include_next <stdlib.h>
              ^~~~~~~~~~
In file included from /Users/travis/Library/Caches/vcpkg/Source/vcpkg/buildtrees/tbb/x64-osx-rel/src/./src/tbb/concurrent_queue.cpp:17:
In file included from /Users/travis/Library/Caches/vcpkg/Source/vcpkg/buildtrees/tbb/x64-osx-rel/src/./include/tbb/tbb_stddef.h:431:
In file included from /Applications/Xcode-10.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/memory:654:
In file included from /Applications/Xcode-10.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/typeinfo:61:
In file included from /Applications/Xcode-10.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/exception:82:
In file included from /Applications/Xcode-10.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/cstdlib:86:
/Applications/Xcode-10.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1/stdlib.h:94:15: fatal error: 'stdlib.h' file not found
#include_next <stdlib.h>
              ^~~~~~~~~~
1 error generated.
make[4]: *** [concurrent_hash_map.o] Error 1
make[4]: *** Waiting for unfinished jobs....
1 error generated.
make[4]: *** [concurrent_queue.o] Error 1
make[3]: *** [tbb] Error 2

-- Configuring done
-- Generating done
-- Build files have been written to: /Users/travis/Library/Caches/vcpkg/Source/vcpkg/buildtrees/tbb/x64-osx-rel
```

**Comment by zhihaoy** [comment]
Root cause found; it's Xcode 10.2's problem:

https://discourse.brew.sh/t/clang-can-no-longer-find-usr-include-header-files-fatal-error-stdlib-h-file-not-found/4523

It is not obvious to vcpkg because vcpkg has to built with GCC, which doesn't reveal Apple toolchain's problem; can we have a warning message in vcpkg to say you need to install SDK headers?
